src/neo/SmartContract/ApplicationEngine.Contract.cs

```diff
@@ -50,6 +50,34 @@ protected internal ContractState CreateContract(byte[] script, byte[] manifest)
             if (!contract.Manifest.IsValid(hash)) throw new InvalidOperationException($"Invalid Manifest Hash: {hash}");
 
             Snapshot.Contracts.Add(hash, contract);
+
+            // Execute _deploy
+
+            ContractMethodDescriptor md = contract.Manifest.Abi.GetMethod("_deploy");
```

Comment:
will NEON support a method named _deploy ?

Reply:
Yes.

Reply:
1. Do we need to call `_deploy` in `System.Contract.Update` syscall?
2. Rename `System.Contract.Create` to `System.Contract.Deploy` ?